channels/il.m3u

```diff
@@ -7,7 +7,7 @@ http://82.80.192.30/shoppingil_ShoppingIL21TVRepeat/_definst_/smil:ShoppingIL21T
 https://kanlivep2event-i.akamaihd.net/hls/live/747600/747600/source1_1.8k/chunklist.m3u8
 #EXTINF:-1 tvg-id="Kan 23-IS" tvg-name="IL: Kan 23" tvg-country="IL" tvg-language="Hebrew" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/6/6b/KanHinuchit.svg/2880px-KanHinuchit.svg.png" group-title="Kids",ch 23 החינוכית
 https://kanlivep2event-i.akamaihd.net/hls/live/747600/747600/source1_2.5k/chunklist.m3u8
-#EXTINF:-1 tvg-id="Channel2News.il" tvg-name="Channel 2 News" tvg-country="IL" tvg-language="" tvg-logo="" group-title="",Channel 2 News
+#EXTINF:-1 tvg-id="Channel2News.il" tvg-name="Channel 2 News" tvg-country="IL" tvg-language="" tvg-logo="" group-title="Local",Channel 2 News
```

Comment:
I'm not sure why it's local and why didn't you change tags for master links then

Reply:
Not quite sure about that tag, chief 
![image](https://user-images.githubusercontent.com/30985701/125194287-df818300-e250-11eb-915f-4bcbfe4fe9b7.png)